docs/cluster.txt

```diff
@@ -1,5 +1,158 @@
-.. _cluster:
+.. _cluster-browser:
 
 ===============
 Cluster Browser
 ===============
+
+The CrateDB admin UI comes with a *cluster browser* that allows you to inspect
+all of the nodes that are in your `cluster`_.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Screenshot
+==========
+
+Here's what a single node cluster looks like:
+
+.. image:: cluster-simple.png
+
+In this example, there is one node named *Monte Civetta*. When selected from
+the left-hand sub-navigation menu, information about a node is displayed.
+
+Features
+========
+
+.. |nbsp| unicode:: 0xA0
+   :trim:
+
+The cluster browser features:
+
+:Node |nbsp| listing:
+
+  The list of nodes gives basic summary information for each node: node name,
+  hostname, CrateDB version, and a visual icon indicating whether the node is a
+  ``master node`` or not and the node :ref:`data status <status-bar>` health
+  check.
+
+:Node |nbsp| list |nbsp| ordering:
+
+  The list of nodes can be ordered by name or node health.
+
+:Detailed node information:
+
+  Selecting a node from the node list will display details about that particular
+  node.
+
+The following subsections explain the detailed node information.
+
+Overview
+--------
+
+:Name:
+
+  The name of the node.
+
+  CrateDB automatically names unnamed nodes.
+
+:Hostname:
+
+  The hostname of the node.
+
+:CrateDB Version:
+
+  The version of CrateDB the node is running.
+
+  This is important information when performing a `rolling upgrade`_.
+
+:REST URL:
+
+  The URL of the node's `HTTP endpoint`_.
+
+:CPU Usage:
+
+  A visual indicator of system CPU utilization.
+
+:Heap Usage:
+
+  A visual indicator of allocated *Java Virtual Machine* (JVM) `heap`_
+  utilization.
+
+:Disk Usage:
+
+  A visual indicator of system disk space utilization.
+
+:CrateDB CPU Usage:
+
+  A visual indicator of allocated CrateDB CPU utilization.
+
+  TODO: is this correct? needs more explaining I think
```

Comment:
@mxm oh, here too

Reply:
This is also available via `sys.nodes`.

Reply:
hmm. are you sure? a few items above this we have "CPU Usage" which I have defined as system CPU usage. from the doc you've linked, wouldn't this be `getProcessCpuLoad`?

Reply:
Sorry, I was looking at the CPU usage above this. You're correct, this should be the process cpu usage.